src/engine/client/ClientApplication.cpp

```diff
@@ -85,6 +85,19 @@ class ClientApplication : public Application {
             // Don't set this for TTY applications as they really aren't DPI aware. Let them scale.
             SDL_SetHint(SDL_HINT_WINDOWS_DPI_AWARENESS, "system");
 #endif
+
+#if defined(__linux__) && defined(BUILD_GRAPHICAL_CLIENT)
+    #if SDL_VERSION_ATLEAST(2, 0, 22)
+            // identify the game by its name in certain
+            // volume control / power control applets,
```

Comment:
Hardcoding the string instead of doing a version check could be better, since then an SDL built with old headers could still work with a newer binary.